modules/lib/lib-portal/src/main/resources/lib/xp/portal.ts

```diff
@@ -526,7 +526,7 @@ export function getMultipartForm(): MultipartForm {
  *
  * @returns {object|null} The named multipart form item.
  */
-export function getMultipartItem(name: string, index: number): MultipartItem | null {
+export function getMultipartItem(name: string, index?: number): MultipartItem | null {
     const bean = __.newBean<MultipartHandler>('com.enonic.xp.lib.portal.multipart.MultipartHandler');
     return __.toNativeObject(bean.getItem(name, index ?? 0));
 }
```

Comment:
I prefer this syntax, as it ties the default value to the variable to the `index` variable.

```suggestion
export function getMultipartItem(name: string, index: number = 0): MultipartItem | null {
    const bean = __.newBean<MultipartHandler>('com.enonic.xp.lib.portal.multipart.MultipartHandler');
    return __.toNativeObject(bean.getItem(name, index));
}
```

Reply:
Yeah, the default value will look better.